src/components/Contacts/ContactListTable.jsx

```diff
@@ -42,110 +27,36 @@ const CustomToolbar = () => (
 const ContactListTable = ({ contacts, deleteContact }) => {
   const [showMessageModal, setShowMessageModal] = useState(false);
   const [messageToField, setMessageToField] = useState('');
+  const theme = useTheme();
+  const isSmallScreen = useMediaQuery(theme.breakpoints.down('sm'));
 
   const handleSendMessage = (contactId) => {
     setShowMessageModal(!showMessageModal);
-    setMessageToField(contactId.value.podUrl);
+    setMessageToField(isSmallScreen ? contactId.podUrl : contactId.value.podUrl);
   };
 
-  const columnTitlesArray = [
-    {
-      field: 'First Name',
-      minWidth: 120,
-      flex: 1,
-      headerAlign: 'center',
-      align: 'center'
-    },
-    {
-      field: 'Last Name',
-      minWidth: 120,
-      flex: 1,
-      headerAlign: 'center',
-      align: 'center'
-    },
-    {
-      field: 'webId',
-      headerName: 'Web ID',
-      minWidth: 150,
-      flex: 1,
-      headerAlign: 'center',
-      align: 'center'
-    },
-    {
-      field: 'Profile',
-      renderCell: (contactData) => <ContactProfileIcon contact={contactData} />,
-      sortable: false,
-      filterable: false,
-      width: 80,
-      headerAlign: 'center',
-      align: 'center'
-    },
-    {
-      field: 'Message',
-      renderCell: (contactId) => (
-        <SendIcon
-          sx={{ color: 'gray', cursor: 'pointer' }}
-          onClick={() => handleSendMessage(contactId)}
+  return (
+    <Box
+      sx={{
+        margin: '20px 0',
+        width: '95vw',
+        height: '500px'
+      }}
+    >
+      {isSmallScreen ? (
+        <ContactListTableMobile
+          contacts={contacts}
+          deleteContact={deleteContact}
+          handleSendMessage={handleSendMessage}
         />
-      ),
-      sortable: false,
-      filterable: false,
-      width: 80,
-      headerAlign: 'center',
-      align: 'center'
-    },
-    {
-      field: 'actions',
-      type: 'actions',
-      headerName: 'Delete',
-      width: 80,
-      getActions: (contactData) => [
-        <GridActionsCellItem
-          icon={<DeleteOutlineOutlinedIcon />}
-          onClick={() => deleteContact(contactData.row.Delete)}
-          label="Delete"
+      ) : (
+        <ContactListTableDesktop
+          contacts={contacts}
+          deleteContact={deleteContact}
+          handleSendMessage={handleSendMessage}
+          // handleProfileClick={handleProfileClick}
```

Comment:
Is this comment out line for profile click temporary while we transition these changes?

Reply:
That was just for testing. I already removed it in a change, just haven't pushed it up yet.